doc/playbook.rst

```diff
@@ -155,13 +155,12 @@ To inspect buckets with the MinIO Client ``mc`` tool, first set up an alias (e.g
     mc alias set usdf-pp https://s3dfrgw.slac.stanford.edu ACCESS_KEY SECRET_KEY
```

Comment:
Should the instructions for AWS credentials also be updated to use a `prompt-processing-dev` profile? I find it less confusing if the profile name matches the user name.